pkg/docker/docker.go

```diff
@@ -445,8 +435,8 @@ func (g generalClient) IsCreated(ctx context.Context, cname string) (bool, error
 	return true, nil
```

Comment:
Why not using container.State.Status (one of "created", "running", "paused", "restarting", "removing", "exited", or "dead")? @gaocegege

Reply:
I expect to follow the implementation in docker-cli.

But cannot remember if the code is copied from docker-cli

Reply:
Got it.

Reply:
Sorry I misunderstand here. The function `IsCreated` returns false if the container cannot be found. Thus I think it works with inspect. WDYT?

Reply:
Since the `created` is one of the status for containers, I think `exist` may reduce ambiguity. WDYT?

Reply:
SGTM

Reply:
https://github.com/tensorchord/envd/issues/478 
I'll resolve the conversation here.